test/Output/VideoTest.php

```diff
@@ -0,0 +1,184 @@
+<?php
+
+namespace Output;
+
+use ClockMock;
+use GetOpt\Option;
+use GetOptMock;
+use GOL\Boards\Board;
+use GOL\Output\Video;
+use PHPUnit\Framework\TestCase;
+
+class VideoTest extends TestCase
+{
+    protected $output;
+    protected $board;
+    protected $getopt;
+
+    protected function setUp(): void
+    {
+        $this->output = new Video();
+        $this->board = new Board(5, 5);
+        $this->getopt = new GetOptMock();
+        exec("rm out/*");
+    }
+
+    /**
+     * @test
+     */
+    public function writeEmptyBoard()
+    {
+        $this->output->checkParameters($this->getopt);
+        $this->output->write($this->board);
+        $this->output->flush();
+
+        $image = imagecreate(5, 5);
+        imagecolorallocate($image, 0, 0, 0);
+        imagecolorallocate($image, 255, 255, 255);
+        imagepng($image, "out/000.png");
+
+        exec("rm in/video.avi");

```

Comment:
Please use php's `unlink()` instead! (its platform agnostic and would also work under windows)
Please adapt this in all tests where you delete files!